openvpn-install.sh

```diff
@@ -181,13 +188,18 @@ hide-version: yes
 use-caps-for-id: yes
 prefetch: yes
 private-address: 10.0.0.0/8
+private-address: fd42:42:42:42::/112
 private-address: 172.16.0.0/12
 private-address: 192.168.0.0/16
 private-address: 169.254.0.0/16
 private-address: fd00::/8
 private-address: fe80::/10
 private-address: 127.0.0.0/8
 private-address: ::ffff:0:0/96' > /etc/unbound/openvpn.conf
+		if [[ "$IPV6_SUPPORT" == 'y' ]]; then
+			echo 'interface: fd42:42:42:42::1
+access-control: fd42:42:42:42::/112 allow' >> /etc/unbound/unbound.conf
```

Comment:
Fix unbound configuration when already installed
```suggestion
access-control: fd42:42:42:42::/112 allow' >> /etc/unbound/openvpn.conf
```